core/src/main/resources/hudson/model/User/sidepanel.jelly

```diff
@@ -32,7 +32,7 @@ THE SOFTWARE.
   <l:side-panel>
     <l:tasks>
       <l:task contextMenu="false" href="${rootURL}/asynchPeople/" icon="icon-up icon-md" title="${%People}"/>
-      <l:task contextMenu="false" href="${rootURL}/${it.url}/" icon="icon-search icon-md" title="${%Status}"/>
+      <l:task contextMenu="false" href="${rootURL}/${it.url}/" icon="symbol-person-circle" title="${%Status}"/>
```

Comment:
I think using the non circle is a better fit, but I will not die on this hill
```suggestion
      <l:task contextMenu="false" href="${rootURL}/${it.url}/" icon="symbol-person" title="${%Status}"/>
```

https://github.com/jenkinsci/jenkins/blob/bdaf2a535e3f3b70b52608fe7fc4f576222adaf2/war/src/main/resources/images/symbols/person.svg 

vs

https://github.com/jenkinsci/jenkins/blob/master/war/src/main/resources/images/symbols/person-circle.svg